src/test/mod.rs

```diff
@@ -629,6 +629,10 @@ fn read_config(filename: &Path) -> (Config, OperationSetting, EmitterConfig) {
         get_config(filename.with_extension("toml").file_name().map(Path::new))
     };
 
+    if !config.was_set().unstable_features() && !is_nightly_channel!() {
+        config.override_value("unstable_features", "true");
+    }
```

Comment:
@topecongiro - know you've already approved the core changes here, but before I merge I want to make sure you are okay with this one check added here in https://github.com/rust-lang/rustfmt/pull/4228/commits/788eb5a03f05ffee154dea6e5a2be11465c63b35 to fix the tests on beta vs adding an inline `// rustfmt-unstable-features: true` comment into each of the hundreds of test files under `tests/{source,target}` that would need them

Reply:
Yeah, that sounds way better to me.